src/app/Plans/components/Wizard/FilterVMs.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Form, FormGroup, TreeView, Title } from '@patternfly/react-core';
+
+// TODO replace with redux or equiv.
+import { VMsOptions } from './mocks/VMs.mock';
+
+interface IFilterVMsProps {
+  Inventory: [];
+}
+
+const FilterVMs: React.FunctionComponent<IFilterVMsProps> = ({ Inventory }: IFilterVMsProps) => {
+  const [activeItems, setItems] = React.useState(VMsOptions);
```

Comment:
@mturley What do you think about inferring types here from the VMsOptions vs defining an interface & conditionally accepting an undefined type? Would we always know the VMsOptions list of types or is this coming from the API where we would need to check for some error when fetching types?

Reply:
+1 for suggestion to move work to implement the correct data structure to a new PR/Issue.